tests/components/smarttub/test_entity.py

```diff
@@ -0,0 +1,18 @@
+"""Test SmartTubEntity."""
+
+from homeassistant.components.smarttub.entity import SmartTubEntity
```

Comment:
Setup the integration and check the state in the state machine instead

https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations

Reply:
That's what I do in `test_climate.py`. This is intended to be a unit test for the base class, which is itself never added to the state machine.

If you prefer, I can just delete this file. The only affect on test coverage would be `entity.py:59`

Reply:
I think its low risk of breakage so let's leave it for now. If another reviewer feels otherwise, please remove it in a followup PR if this merges before they comment.